src/qiskit_toqm/toqm_plugin.py

```diff
@@ -0,0 +1,62 @@
+from qiskit.transpiler.preset_passmanagers.plugin import PassManagerStagePlugin
+from qiskit.transpiler.preset_passmanagers import common
+from qiskit_toqm import *
```

Comment:
I'm surprised pylint doesn't complain about this, normally it really doesn't like `*` imports

Reply:
I don't think we actually have pylint set up for this project. There's a `nox` lint script set up that came from the scikit-build template, but it isn't enforced by CI currently, and it hasn't been run in a while. It'd be nice to get that sorted in a separate PR.